test/new-e2e/tests/agent-platform/install/installparams/install_params.go

```diff
@@ -62,3 +64,24 @@ func WithFlavor(flavor string) Option {
 		p.Flavor = flavor
 	}
 }
+
+// WithUpgrade specify if the upgrade environment variable is used when installing the agent
+func WithUpgrade(upgrade bool) Option {
+	return func(p *Params) {
+		p.Upgrade = upgrade
+	}
+}
+
+// WithCustomAPIKey specify a custom api key to use when installing the agent
+func WithCustomAPIKey(apiKey string) Option {
+	return func(p *Params) {
+		p.CustomAPIKey = apiKey
+	}
+}
+
+// WithCustomPipelineID specify a custom api key to use when installing the agent
```

Comment:
I think you forgot to change the comment after copy/paste. 
I'd name the function `WithPipelineID` since the attribute is `PipelineID`